linters/groovy/.groovylintrc.json

```diff
@@ -2,13 +2,19 @@
   "extends": "recommended",
 
   "rules": {
+    "convention.CompileStatic": "off",
+
+    "dry.DuplicateNumberLiteral": { "enabled": false },
+    "dry.DuplicateStringLiteral": { "enabled": false },
+
     "formatting.LineLength": { "length": 140 },
     "formatting.Indentation": { "spacesPerIndentLevel": 1 },
     "formatting.NoTabCharacter": "off",
 
-    "size.ParameterCount": { "maxParameters": 9 },
+    "InvertedCondition": "off",
```

Comment:
how many warnings was this?

Reply:
There are about 8 of them I think.
I disable it here since I thought this is the standard?

Reply:
i think we have this enabled for JS but check with @gimre-xymcity